power_measurement.adoc

```diff
@@ -111,93 +107,110 @@ If you are measuring the performance of an experimental framework or system, you
-* No benchmark- or benchmarking-specific hacks
-* Any changes in power management behavior must not have manual intervention or have awareness of the benchmark.
+A power meter configuration must be reported in a file called
+`analyzer_table.*` placed as follows:
+
+- If the configuration is common to all scenarios, benchmarks and systems: under the `<division>/<submitter>/measurements` directory.
+
+- If the configuration is common to all scenarios and benchmarks running on a system: under the `<division>/<submitter>/measurements/<system>` directory.
+
+- If the configuration is common to all scenarios for a benchmark running on a system: under the `<division>/<submitter>/measurements/<system>/<benchmark>` directory.
+
+- If the configuration is specific to a scenario for a benchmark running on a system: under the `<division>/<submitter>/measurements/<system>/<benchmark>/<scenario>` directory.
 
-== Power Logs
+The file format must be human readable, for example, Markdown (recommended), TXT, CSV or HTML.
+Examples: https://github.com/mlcommons/inference_results_v1.1/blob/main/closed/Qualcomm/measurements/r282_z93_q8-qaic-v1.5.9-aic100/analyzer_table.md[`md`] (single channel),
+https://github.com/mlcommons/inference_results_v1.1/blob/main/closed/Dell/power/R750xa_A100-PCIE-40GBx4_analyzer_table.md[`md`] (multi-channel),
+https://github.com/mlcommons/inference_results_v1.1/blob/main/closed/NVIDIA/results/AGX_Xavier_TRT_MaxQ/AGX_Xavier_TRT_MaxQ.csv[`csv`] (single channel),
+https://github.com/mlcommons/inference_results_v1.1/blob/main/closed/NVIDIA/results/A100-PCIex8_TRT_MaxQ/A100-PCIex8_TRT_MaxQ.csv[`csv`] (multi-channel).
 
-Power logs will need to be submitted. All logs created as part of Power
-measurement will need to be submitted including the power analyzer
-ranging and the performance measurement.
+### Power Management Settings
 
-Power Logs are generated by the software running on the Director.
+For reproducibility and verifiability of each submission, the Power Management
+Settings used to achieve the measured performance and power for the
+submission must be documented.
 
-== Loadgen
+The Power Management Settings must be reported in a file called
+`power_settings.*` placed according to the same rules as for the Power Analyzer
+Settings.
 
-The flow for power uses the same Loadgen as used for the performance
-runs. No additions are being made. Power flow uses the start and stop
-timestamp given by the loadgen for synchronizing the performance section
-of the benchmark and uses these markers for anchoring the window in
-which power is measured.
+Examples: https://github.com/mlcommons/inference_results_v1.1/blob/main/closed/Qualcomm/measurements/g292_z43_q16-qaic-v1.5.6-aic100/power_settings.md[`md`], 
+https://github.com/mlcommons/inference_results_v1.1/blob/main/closed/Dell/power/XE8545_A100-SXM-80GBx4_power_settings.md[`md`].
```

Comment:
The Dell filename is not conformant to the pattern `power_settings.*` you listed above.

Reply:
You are right. The location is also off. That's why we should standardise these things. It's a good example otherwise, but I'm happy to other suggestions.